src/styles/sidebar/components/MarkdownEditor.scss

```diff
@@ -28,26 +28,24 @@
 }
 
 .MarkdownEditor__toolbar-button {
-  @include buttons.reset-native-btn-styles;
-  @include layout.row(center, center);
-  appearance: none;
+  @include buttons.button--icon-only($with-active-state: false);
   min-width: 24px;
   min-height: 24px;
 
-  color: var.$grey-5;
+  padding-top: 0;
+  padding-bottom: 0;
 
-  &:hover,
-  &:focus {
-    color: var.$grey-7;
-  }
+  color: var.$grey-5;
+  transition: none;
 
   &:disabled {
     color: var.$grey-3;
   }
+}
 
-  &-icon {
-    @include utils.icon--xsmall;
-  }
+svg.MarkdownEditor__toolbar-button-icon {
```

Comment:
I believe this rule could be nested under `.MarkdownEditor__toolbar-button` and the selector could be:

`&svg`

but don't quote me on that. Just a thought.

Reply:
roger!

Reply:
I could not get that to work, but this will

svg {
   ...
}

However, that breaks our pattern of styling svg's via classname and makes MarkdownEditor__toolbar-button-icon classname irrelevant. I think we just leave this or make change to the button mixin so it won't set the icon size. (or perhaps can be overridden)

Reply:
Sure, leave it. NBD.